### PR TITLE
Update to latest rails

### PR DESCRIPTION
**Pull request by nviennot**

### CODE REVIEW

File: config/secrets.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: 9f147bb0e4e15668c47f2d79ae6114442270689156132e913d59c262350703eb02a447c9bb09bd07a3b8049f030158cd26ac180cda839baf0f67ff9158a75b28
```
Comment (deontologician) [inline config/secrets.yml]: should this be here?

Comment (nviennot) [inline config/secrets.yml]: yeah, it needs something, it's to sign cookies. We can also put `<%= SecureRandom.hex %>` if it's too weird (but cookies would be invalidated at each restart of the server -- but it's not like we are using any of these features).

Comment (deontologician) [inline config/secrets.yml]: I'd be a fan of using `<%= SecureRandom.hex %>` if that will work, that way no one is tempted to use a default key that's shared publicly

**Comment by deontologician** [comment]
Other than the key, looks good to me

**Comment by nviennot** [comment]
I've updated the key :)

**Comment by deontologician** [comment]
thanks again!

**Pull request merged**